bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -419,6 +419,9 @@ impl Height {
     /// The maximum absolute block height.
     pub const MAX: Self = Height(LOCK_TIME_THRESHOLD - 1);
 
+    /// Creates a new instance having its default value.
```

Comment:
```suggestion
    /// Creates a new zero instance, equivalent to `Time::ZERO`.
```

Reply:
Oh yes, I just forget to correct them . My Bad

Reply:
and What about the `Time` struct -> IMO its new instance  should be `Time::MIN` not `Time(0)` ?

Reply:
@15IITian oo good call. Yes, should be `Time::MIN`, and now that I think about it,  I don't think it should have `Default` since this value can't represent 0, and the minimum value is a weird arbitrary date. (I'm curious if you know what this date is, offhand.)

Given that you can't actually represent the full range of UNIX times with this type ... maybe you shouldn't be using it in bdk as a "UNIX timestamp" type, so its lack of `Default` won't be such a burden to you.

But I think that `absolute::LockTime` and `relative::LockTime` should both have `ZERO` constants which correspond to 0 blocks and the consensus-encoded zero `u32`. Unfortunately I also don't think that `Default` is appropriate there because the type is just a bit too abstract. (In Rust it's really unclear what this trait is supposed to mean, but for "integer" types there is at least a convention that it equals 0. The locktime types I would argue aren't "integer" types.)

Reply:
Never mind me, the minimum time *is* 0 (which is the "arbitrary date" Jan 1 1970, but whatever). When we convert `Time` to a `LockTime` type is when all the weird offsetting happens.

So, yes, you are correct that we should use `Time::MIN` for the `new`/`default` value, but it is fine to use these as a default.